scio-core/src/main/scala/com/spotify/scio/coders/DerivedCoder.scala

```diff
@@ -41,7 +41,7 @@ private object Derived extends Serializable {
       i = i + 1
     }
 
-    val stack = CoderException.prepareStackTrace
+    lazy val stack = CoderException.prepareStackTrace
```

Comment:
Why did you change that ?

Reply:
I was trying to delay the stack trace preparation to see if the speed improves. its negligible, shall i remove this?

Reply:
Yes. The purpose is to pre-generate the stack trace to display the call site instead of beam's internal when an exception occurs. Maky it lazy is a bug.

Reply:
I see now. Thanks for explaining.